### PR TITLE
work and form feature 

### PR DESCRIPTION
**Pull request by 0zedd**
add form to html stlyed with css
add background image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,9 +207,39 @@ <h4 class="sec_abt">
...
-                
-
             </div>
     </section>
+
+
+    <footer class="reach_out">  
+            <h2 class="cont_me">Contact me</h2>
+            <P class="cont_input">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
```
Comment (tufoinnkuo10) [inline index.html:217]: - [x] please use the same font type like the Figma version

File: index.html [hunk 2]
```diff
@@ -209,9 +207,39 @@ <h4 class="sec_abt">
...
+                        <input class="username" type="text" name="username" placeholder="username" required maxlength="30">
+                    </li>
+                    <li>
+                        <input class="email" type="email" name="email_address" placeholder="domain@gmail.com" required>
+                    </li>
+                    <li>
+                        <textarea class="comment" name="comments" placeholder="Write your message here" required maxlength="500"></textarea>
+                    </li>
+                    <li class="submit">
+                        <button type="submit">Get It Touch</button>
```
Comment (tufoinnkuo10) [inline index.html:231]: - [x] kindly add the hover interaction on the button

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @0zedd 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Form submits successfully


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by muneebulrehman — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved ✔️

## Highlights
- Good follow-up with the previous reviewer.
- Good HTML and CSS

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**